injected/playwright.config.js

```diff
@@ -28,6 +28,7 @@ export default defineConfig({
             name: 'apple',
             // prettier-ignore
             testMatch: [
+                'integration-test/navigator-interface-insecure.js',
```

Comment:
For now, just adding it where a bug was introduced recently. Upcoming platform tests will (hopefully) make this kind of test redundant, so there's no need for more platforms here for now.

Reply:
If the cost is low to add other platforms, I'd do that.